src/buildercore/cloudformation.py

```diff
@@ -20,3 +21,29 @@ def validate_template(pname, rendered_template):
     "remote cloudformation template checks."
     conn = core.boto_conn(pname, 'cloudformation', client=True)
     return conn.validate_template(TemplateBody=rendered_template)
+
+# CloudFormation is nicely chopped up into:
```

Comment:
just moving things progressively here from `cfngen.py`, no code change apart from the class name